zkstack_cli/crates/zkstack/build.rs

```diff
@@ -19,3 +29,50 @@ fn main() -> eyre::Result<()> {
     .unwrap();
     Ok(())
 }
+
+fn copy_completion_scripts() -> anyhow::Result<()> {
+    let crate_name = env!("CARGO_PKG_NAME");
+
+    // Create local config directory
+    let local_config_dir = config_local_dir().unwrap().join(crate_name);
+    std::fs::create_dir_all(&local_config_dir)?;
+
+    // Array of supported shells
+    let shells = ["bash", "zsh"];
+
+    // Copy completion files
+    let completion_dir = local_config_dir.join("completion");
+    std::fs::create_dir_all(&completion_dir)?;
+
+    for shell in &shells {
+        let completion_file = format!("_{}_{}", crate_name, shell);
+        std::fs::copy(
+            format!("completion/{}", &completion_file),
+            completion_dir.join(&completion_file),
+        )?;
+
+        // Source the completion file inside .{shell}rc
+        let shell_rc = home_dir()
+            .context("missing home directory")?
+            .join(format!(".{}rc", shell));
+
+        if shell_rc.exists() {
+            let shell_rc_content = std::fs::read_to_string(&shell_rc)?;
+
+            if !shell_rc_content.contains("# zkstack completion") {
+                let completion_path = completion_dir.join(&completion_file);
+                let completion_path = completion_path.to_str().unwrap();
+
+                std::fs::write(
+                    shell_rc,
+                    format!(
+                        "{}\n# zkstack completion\nsource \"{}\"\n",
+                        shell_rc_content, completion_path
+                    ),
+                )?;
+            }
+        }
```

Comment:
So, you never update it, once it set, right?

Reply:
Yes, it's enough to update the autocompletion files in the relative folders 👍